packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -73,6 +73,8 @@ const router = mesh.addVirtualRouter('router', {
 ```
 
 The router can also be created using the constructor and passing in the mesh instead of calling the `addVirtualRouter()` method for the mesh.
+It is important to note when calling the `addVirtualRouter()` method for the mesh, router is created in the same stack where the mesh is called.
+On the other hands, when the router is created using the constructor, it is created in the current stack.
 The same pattern applies to all constructs within the appmesh library, for any mesh.addXZY method, a new constuctor can also be used.
```

Comment:
Maybe a code example illustrating the previous sentence would be helpful?

Reply:
Instead of adding an example, I added the link to `Adding VirtualNode` and `Adding VirtualGateway` header.